### ISSUE TITLE
Mysql: Unhandled exception: "unpack requires a buffer of 4 bytes"  

### ISSUE THREAD

**Issue by raphaelquati** [microsoft/pgtoolsservice#335]
Azure Data Studio: 1.40.0-insider
Mysql Plugin: 0.2.2
pgtoolsservice: 1.6.1
MySql Server: 5.7.40 (Docker)

Select * from T1 gives the error:

```
[Error - 00:11:28] Request query/subset failed.
  Message: Unhandled exception while handling request method query/subset: "unpack requires a buffer of 4 bytes"
  Code: -32603 
```

The table structure:

```sql
CREATE TABLE `T1` (
  `c1` int(11) NOT NULL AUTO_INCREMENT,
  `c2` longtext,
  `c3` tinyint(1) NOT NULL,
  `c4` varchar(100) CHARACTER SET utf8 DEFAULT NULL,
  `c5` tinyint(1) NOT NULL,
  `c6` int(11) NOT NULL,
  `c7` longtext,
  `c8` longtext,
  `c9` longtext,
  `c10` longtext,
  `c11` varchar(100) CHARACTER SET utf8 DEFAULT NULL,
  `c12` varchar(100) CHARACTER SET utf8 DEFAULT NULL,
  `c13` varchar(100) CHARACTER SET utf8 DEFAULT NULL,
  `c14` varchar(100) DEFAULT NULL,
  `c15` bit(1) NOT NULL DEFAULT b'0',
  PRIMARY KEY (`c1`)
) ENGINE=InnoDB AUTO_INCREMENT=131 DEFAULT CHARSET=latin1
```

**Comment by rishky-msft** [comment]
Hi @raphaelquati 

You can try this vsix on windows and confirm if the issue is fixed for you.
https://github.com/microsoft/azuredatastudio-mysql/releases/download/v0.2.3-alpha.1/azuredatastudio-mysql-0.2.3-alpha.1.vsix

**Comment by raphaelquati** [comment]
Hi @rishky-msft ,

This version fix the error. Thanks!

In time:
 I cannot test anymore.... I 've updated the Azure Data Studio Preview to the last version, and nothing is working now, even mssql.....
![image](https://user-images.githubusercontent.com/815086/197286879-bc0cd9e2-4cc6-4e37-b246-84dd52ed07aa.png)

**Comment by rishky-msft** [comment]
The changes will be available in the next release 0.2.3 of
https://github.com/microsoft/azuredatastudio-mysql
